src/PowerShell/Utilities/ScriptBuilder.ts

```diff
@@ -9,6 +9,9 @@ export default class ScriptBuilder {
         let command = `Clear-AzContext -Scope Process;
              Clear-AzContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue;`;
         if (scheme === Constants.ServicePrincipal) {
+            if (args.environment == "AzureStack") {
+                command += `Add-AzEnvironment -Name ${args.environment} -ARMEndpoint ${args.resourceManagerEndpointUrl} | out-null;`;
```

Comment:
Do we not need keyvault and storage suffixes as in the case of Az CLI ?

Reply:
if we are not doing any storage/keyvault data operations , we dont need them

Reply:
We don't want to put any restriction on the Az powershell script, so it's better if I set it here as well.

Reply:
we don't need to set storage/keyvault suffixes for Az PowerShell here. these are already set correctly by default when -ARMEndpoint parameter is passed.